changes/2103.bugfix.rst

```diff
@@ -0,0 +1 @@
+Cookiecutter has been improved to escape special characters & < and >.
```

Comment:
Cookiecutter is a standalone project, not part of Briefcase; we're adding a specific filter, not fixing Cookiecutter itself.
```suggestion
A Cookiecutter filter for escaping XML content has been added.
```

Also - although this change in service of a fixing a bug, this particular change is adding a feature - a new cookiecutter filter for quoting XML content. As such, the file should be `2103.feature.rst`.